src/webui/api/torrentscontroller.cpp

```diff
@@ -320,11 +395,23 @@ void TorrentsController::trackersAction()
         case BitTorrent::TrackerEntry::NotWorking:
             status = tr("Not working"); break;
         }
-        trackerDict[KEY_TRACKER_STATUS] = status;
-        trackerDict[KEY_TRACKER_PEERS] = data.numPeers;
-        trackerDict[KEY_TRACKER_MSG] = data.lastMessage.trimmed();
 
-        trackerList.append(trackerDict);
+        trackerList << QVariantMap {
+            {KEY_TRACKER_URL, tracker.url()},
+            {KEY_TRACKER_TIER, tracker.tier()},
+            {KEY_TRACKER_STATUS, status},
+            {KEY_TRACKER_PEERS_COUNT, data.numPeers},
+            {KEY_TRACKER_MSG, data.lastMessage.trimmed()},
+#if LIBTORRENT_VERSION_NUM >= 10000
```

Comment:
Do we support libtorrent < 1.0?

Reply:
>Do you know?

We don't support libtorrent < 1.0.
I just thought you had the wrong version number here...

Reply:
This was copied over from `trackerlistwidget.cpp`